pkg/apis/core/types.go

```diff
@@ -3977,7 +3977,8 @@ type LocalObjectReference struct {
 // TypedLocalObjectReference contains enough information to let you locate the typed referenced object inside the same namespace.
 type TypedLocalObjectReference struct {
 	// APIGroup is the group for the resource being referenced
-	APIGroup string
+	// +optional
+	APIGroup *string
```

Comment:
Add comments about this optional field.
Discussed with @thockin, if APIGroup is not specified, the reference Kind must be PersistentVolumeClaim. For any other third-party types, APIGroup is required. This is alpha feature gated and may change in the future.

Reply:
Done.